### ISSUE TITLE
A11y_.NET Core_WPF_EditExaminer_ImmediateWindow_Resize: At 200% zoom, "Immediate Window" control name is getting truncated.

### ISSUE THREAD

**Issue by v-inanda** [dotnet/wpf#7873]
### Description

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[WPF]_Win32_May23;#.NET Core;#A11ySev2;#WCAG1.4.4;#DesktopApp;#Win11;#Zoom;#Benchmark;

### Environment Details:
Application Name: .NET Core
.NET Version: .NET-SDK-8.0.100-preview.4.23260.5
WPF
Operating System: Windows 11 Enterprise 21H2

### Pre-Requisites:

1. Open settings, tab till Display and press enter.
2. Tab till Scale and press enter.
3. Tab till Text Size and press enter.
4. Zoom the screen to 200% using Text Size.

### Reproduction Steps

1. Launch VS 2022 Preview
2. Navigate to Sample Applications and right click on the "Editing Examiner Demo" and click on build.
3. Then right click on Editing Examiner Demo then Click on Debug-->Start New Instance. 
4. "Edit Examiner" screen should open.
5. Zoom the screen to 200% using Text Size and observe that "Immediate Window" control name is getting truncated in 200% zoom.

### Expected behavior

At 200% zoom, "Immediate Window" control name should not get truncated. "Immediate Window" text should completely visible.

### Actual behavior

At 200% zoom, "Immediate Window" control name is getting truncated. The text is not completely visible.

![A11y_ NET Core_WPF_EditExaminer_ImmediateWindow_Resize](https://github.com/dotnet/wpf/assets/91525824/db8caa48-31af-4ea8-ad8c-9d67089dbd6e)

### Impact

Users with low vision who zoom the page for better visibility will face difficulty in understanding what is the title of the Control if "Immediate Window" text is not visible.

**Comment by harshit7962** [comment]
Updated the code to fix the issue. Kindly fetch the new code and test for the same.
Merged PR at: https://github.com/microsoft/WPF-Samples/pull/494

**Comment by v-inanda** [comment]
@harshit7962 Verified the issue and it is still reproducing.

![image](https://github.com/dotnet/wpf/assets/91525824/a6c3453e-8af0-4239-840a-32c0ce520a42)

**Comment by v-inanda** [comment]
@harshit7962 Verified the issue and it is fixed. Hence, closing the bug.

![200](https://github.com/dotnet/wpf/assets/91525824/ccc97b37-f0c1-4801-8b8e-1e836a84af81)
